Ambit/Source/Ambit/Mode/ConfigImportExport.spec.cpp

```diff
@@ -432,14 +432,12 @@ void ConfigImportExportSpec::Define()
                 setBuckets.Add("BucketName");
                 return setBuckets;
             };
-
-            Exporter->SetMockS3ListBuckets(MockListBuckets);
+            SetMockS3ListBuckets(MockListBuckets);
```

Comment:
Referring to this method in an unqualified way like this is confusing to comprehend and potentially dangerous. Refactor so that it's `Exporter->SetMockS3ListBuckets()` as it was before. Same goes for other mocking setter methods.